README.md

```diff
@@ -32,6 +32,9 @@ $ cp .env.template .env  # (first time only)
 
 The `.env` file is used by flask to set environment variables when running `flask run`. This enables things like development mode (which also enables features like hot reloading when you make a file change). There's also a [SECRET_KEY](https://flask.palletsprojects.com/en/1.1.x/config/#SECRET_KEY) variable which is used to encrypt the flask session cookie.
 
+## Add Authentification
+Add API key, token and Trello board id to auth.py file in repository root directory
```

Comment:
This instruction sounds like you're assuming the reader knows about Trello and that the app uses it.